src/osd/PGLog.h

```diff
@@ -1270,24 +1270,23 @@ struct PGLog : DoutPrefixProvider {
        ldpp_dout(dpp, 20) << "update missing, appended " << *p << dendl;
        roll_forward_to = p->version;
       }
-      if (p->soid <= last_backfill &&
-	  !p->is_error()) {
-	if (missing.may_include_deletes) {
-	  missing.add_next_event(*p);
-	} else {
-	  if (p->is_delete()) {
-	    missing.rm(p->soid, p->version);
-	  } else {
-	    missing.add_next_event(*p);
-	  }
-	  if (rollbacker) {
-	    // hack to match PG::mark_all_unfound_lost
-	    if (maintain_rollback && p->is_lost_delete() && p->can_rollback()) {
-	      rollbacker->try_stash(p->soid, p->version.version);
-	    } else if (p->is_delete()) {
-	      rollbacker->remove(p->soid);
-	    }
-	  }
+      if (p->soid <= last_backfill && !p->is_error()) {
+        if (missing.may_include_deletes) {
+          missing.add_next_event(*p);
+        } else {
+          if (p->is_delete()) {
+            missing.rm(p->soid, p->version);
+          } else {
+            missing.add_next_event(*p);
+          }
+          if (rollbacker) {
+            // hack to match PG::mark_all_unfound_lost
+            if (maintain_rollback && p->is_lost_delete() && p->can_rollback()) {
+              rollbacker->try_stash(p->soid, p->version.version);
+            } else if (p->is_delete()) {
+              rollbacker->remove(p->soid);
+            }
+          }
 	}
```

Comment:
Is this patch just a whitespace change?  If so, please change the commit message to something like "osd/PGLog: clean up spacing in append_log_entries_update_missing".  If not, indicate what changed and to what purpose.

Reply:
yes, updated commit